src/to_xml.jl

```diff
@@ -96,8 +96,10 @@ function to_xml(ts::AbstractTestSet)
     total_ntests = 0
     total_nfails = 0
     total_nerrors = 0
-    x_testcases = map(ts.results) do result
+    x_testcases = map(enumerate(ts.results)) do (i, result)
         x_testcase, ntests, nfails, nerrors = to_xml(result)
+        set_attribute!(x_testcase, "classname", ts.description)
+        set_attribute!(x_testcase, "name", x_testcase["name"] * " ($i)")
```

Comment:
Because its only `Pass`es that have the same `name`, we could add in a check before updating the attribute.

And we can also use `total_ntests` to avoid needing to use enumerate. You'll need to move it to below `total_ntests += ntests`.

```suggestion
        isa Pass && set_attribute!(x_testcase, "name", x_testcase["name"] * " (Test $total_ntests)")
```